src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java

```diff
@@ -46,11 +46,11 @@
 public abstract class DiseaseAnnotation extends Association {
 
 	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer")
-	@KeywordField(name = "allianceCurie_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
+	@KeywordField(name = "diseaseAnnotationCurie_keyword", aggregable = Aggregable.YES, sortable = Sortable.YES, searchable = Searchable.YES, normalizer = "sortNormalizer")
 	@Column(unique = true)
 	@JsonView({View.FieldsOnly.class})
 	@EqualsAndHashCode.Include
-	private String allianceCurie;
+	private String diseaseAnnotationCurie;
```

Comment:
@markquintontulloch is there a migration in the DB for this?

Reply:
good point - will add one

Reply:
@oblodgett - there is now!